### PR TITLE
Fix missing scrollbar in xml viewer

### PR DESCRIPTION
**Pull request by LaviniaStiliadou**
Should fix issue #79.

### CODE REVIEW

File: components/bpmn-q/modeler-component/QuantumWorkflowModeler.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ import {
 import * as editorConfig from "./editor/config/EditorConfigManager";
 import { initEditorEventHandler } from "./editor/events/EditorEventHandler";
 import $ from "jquery";
-import { edit } from "ace-builds";
+
+/* global ace */
```
Comment (mbeisel) [inline components/bpmn-q/modeler-component/QuantumWorkflowModeler.js:34]: shouldn't this be removed?

**Review by mbeisel — CHANGES_REQUESTED** [review]
At the bottom of the Workflow modeler there is a whitespace - when opening the XML-Viewer it is even larger than usual. Can the whitespace be removed altogether or at least remain the same when opening the XML viewer?
grafik

File: components/bpmn-q/modeler-component/QuantumWorkflowModeler.js (hunk 1 above)
Comment (LaviniaStiliadou) [reply components/bpmn-q/modeler-component/QuantumWorkflowModeler.js]: Great point, I removed it

**Review by wederbn — APPROVED** [review]
LGTM

**Pull request merged**